packages/react/src/dropdown-menu/DropdownMenu.test.tsx

```diff
@@ -41,13 +52,15 @@ function renderBasicDropdownMenu() {
 }
 
 describe('DropdownMenu', () => {
```

Comment:
These tests needed to be updated due to the fact that we've switched to conditional rendering